content/en/docs/reference/glossary/Quantity.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Quantity
+id: quantity
+date: 2023-01-24
+short_description: >
+ Quantity contains GB and GiB. 
+
+aka: 
+tags:
+- fundamental
+---
+
+Quantity refers to a specific amount of a resource, such as CPU or memory, that is allocated to a Pod in which a container is running. 
```

Comment:
Hi, this isn't right.

Have a look at (but _don't_ plagiarize) the callout **Hold up, what's a Quantity?** in https://book.kubebuilder.io/cronjob-tutorial/api-design.html

Reply:
A Quantity represents a [dimensionless](https://en.wikipedia.org/wiki/Dimensionless_quantity) number (the field name should say what it's a quantity of).

Reply:
I got your point but I dont understand why you dont want me to plagiarize?   this documentation is written in a simple and easy words. I dont think my sentence would be that much clear. WDYT?

Reply:
Should I remove the **specific** from that?

Reply:
Try to write in your own words rather than completely copy-paste. You can take a reference from the resources @sftim has shared above.

Reply:
got it.